packages/ember/tests/routing/toplevel_dom_test.js

```diff
@@ -1,56 +1,11 @@
-import { run } from 'ember-metal';
-import { compile } from 'ember-template-compiler';
-import { Application } from 'ember-application';
-import { jQuery } from 'ember-views';
-import { NoneLocation } from 'ember-routing';
-import { setTemplates, setTemplate } from 'ember-glimmer';
+import { moduleFor, ApplicationTestCase } from 'internal-test-helpers';
 
-let App, templates, container;
+moduleFor('Top Level DOM Structure', class extends ApplicationTestCase {
```

Comment:
@josemarluedke It is ok for this test, but I want to mention there is a difference in how the application is booted here you may not have noticed.

In the original, options for `Application.create` are only `name, rootElement`. If you look at [`AbstractApplicationTestCase`](https://github.com/emberjs/ember.js/blob/2f686e30b65d52b2c48a8f6fd27df199cd143a62/packages/internal-test-helpers/lib/test-cases/abstract-application.js#L30) you'll see that it passes `autoboot: false`. The old tests were implicitly testing an auto-booted app, but using `deferReadiness` and `advanceReadiness` to provide a window during which they could set things up. Using `ApplicationTestCase`, you're instead using `autoboot: false` and [a call to the `visit` API](https://github.com/emberjs/ember.js/blob/2f686e30b65d52b2c48a8f6fd27df199cd143a62/packages/internal-test-helpers/lib/test-cases/abstract-application.js#L58) to do something similar.

As long as the semantics of autoboot aren't the subject of the test (I don't think they are here) this is good to go 👍